proposals/0065-issue-pr-label-system.md

```diff
@@ -0,0 +1,254 @@
+- **`triage/needs-information`**: Issue needs more details from the author before it can be properly evaluated or worked on.
+
+**Workflow**: All issues start with `triage/needs-triage`. During triage, maintainers either replace this with another triage label or remove it entirely. The absence of any triage label (along with issue comments) indicates that triage action has been taken, such as declining or marking as duplicate.
+
+#### 2. Review Labels
+
+These labels manage the pull request review lifecycle:
+
+- **`review/needs-review`**: Initial state for all new PRs. Indicates the PR needs review from maintainers.
+- **`review/changes-requested`**: Reviewers have requested changes. Author action is needed.
+- **`review/lgtm`**: "Looks Good To Me" - PR has received technical approval from a reviewer with domain expertise.
+- **`review/approved`**: PR has received final approval from a maintainer with merge rights (per OWNERS or equivalent).
+
+**Workflow**: All PRs start with `review/needs-review`. After review, this transitions to either `review/changes-requested` or `review/lgtm`. Once approved by appropriate maintainers, it becomes `review/approved`.
+
+#### 3. Blocking Labels
+
+These labels indicate that an issue or PR cannot proceed:
+
+- **`blocked/needs-decision`**: Waiting for architectural or design decision
+  from maintainers or community.
+- **`blocked/hold`**: Explicitly placed on hold by a maintainer. Should not be merged or closed even if otherwise ready.
+
+**Workflow**: These labels can be applied at any time when the item becomes blocked. They should be removed when the blocking condition is resolved.
+
+#### 4. Lifecycle Labels (Optional)
+
+These labels track the activity status of issues and PRs. This category is optional and would only be implemented if automated lifecycle management is adopted in the future:
+
+- **`lifecycle/stale`**: No activity for an extended period (e.g., 90 days). Candidate for closure if no activity resumes.
+- **`lifecycle/frozen`**: Should not be auto-closed due to inactivity. Reserved for important long-running items.
+- **`lifecycle/rotten`**: No activity after being marked stale (e.g., 30 additional days). Will be auto-closed soon.
+
+**Workflow**: These would typically be managed by automation based on activity timestamps. Currently, Prometheus projects do not use automated lifecycle management, so these labels are reserved for potential future use.
+
+### Label Interactions
+
+Labels from different categories work together:
+
+- **Active PR ready for review**: `review/needs-review` (no blocking labels)
+- **PR waiting on author**: `review/changes-requested`
+- **PR approved but held**: `review/approved` + `blocked/hold`
+- **Issue accepted and ready**: `triage/accepted` (no blocking labels)
+- **Issue needing more info**: `triage/needs-information`
+- **Issue declined or duplicate**: No triage label (removed after triage with explanatory comment)
+
+### Existing Label Taxonomies
```

Comment:
Maybe this section should have stronger recommendations how to deal with the existing labels? Most of these seem useful to keep using on a best effort basis, but it might also be confusing to have so many labels, so we might want to be a bit stricter what to use and what to kick out.

Reply:
Edited, wdyt?